drivers/input/rmi4/rmi_f21.c

```diff
@@ -32,14 +32,14 @@ static irqreturn_t rmi_f21_attention(int irq, void *ctx)
 	struct f21_data *f21 = dev_get_drvdata(&fn->dev);
 	struct rmi_driver_data *drvdata = dev_get_drvdata(&fn->rmi_dev->dev);
 	int error;
-	bool bpressed = false;
+	bool pressed = false;
 
 	if (drvdata->attn_data.data) {
 		if (drvdata->attn_data.size < f21->attn_data_size) {
 			dev_warn(&fn->dev, "f21 interrupted, but data is missing\n");
 			return IRQ_HANDLED;
 		}
-		bpressed = (((u8 *)drvdata->attn_data.data)[f21->max_number_Of_finger] &
+		pressed = (((u8 *)drvdata->attn_data.data)[f21->max_number_Of_finger] &
```

Comment:
We should use attn data size -1 here right?  otherwise we'll get wrong index in old forcepad.